pkg/controller/migmigration/annotation.go

```diff
@@ -217,7 +217,7 @@ func (t *Task) annotatePods(client k8sclient.Client) error {
 			if pod.Labels == nil {
 				pod.Labels = make(map[string]string)
 			}
-			pod.Labels[IncludedInStageBackupLabel] = t.UID()
+			pod.Labels[IncludedInStageBackupLabel] = t.StagePodSearchLabel()
```

Comment:
I get the method to create the label value is for consistency but thinking just `t.UID() + "-pod"` clearer.